frontend/views/containers/contributions/PaymentMethods.vue

```diff
@@ -1,66 +1,122 @@
-        span.has-text-1.c-description {{method.description}}
+        button.is-icon-small.is-btn-shifted(
+          type='button'
+          :aria-label='L("Remove method")'
+          @click='removeMethod(index)'
+          data-test='remove'
+        )
+          i.icon-times
+
+  button.link.has-icon(
+    type='button'
+    @click='handleAddMethod'
+    data-test='addMethod'
+  )
+    i.icon-plus
+    i18n Add more
 </template>
 
 <script>
-import Tooltip from '@components/Tooltip.vue'
-import SvgBitcoin from '@svgs/bitcoin.svg'
-import SvgMoney from '@svgs/money.svg'
+import { mapGetters } from 'vuex'
+import Vue from 'vue'
 import L from '@view-utils/translations.js'
 
 export default {
-  name: 'PaymentsMethod',
-  components: {
-    SvgBitcoin,
-    SvgMoney,
-    Tooltip
-  },
-  props: {
-    selected: {
-      type: String,
-      required: true,
-      validator: (method) => ['manual', 'bitcoin'].includes(method)
+  name: 'PaymentMethods',
+  components: {},
+  data: () => ({
+    config: {
+      options: {
+        // key to store on state and corresponding translation (if needed)
+        bitcoin: 'Bitcoin',
+        paypal: 'Paypal',
+        venmo: 'Venmo',
+        other: L('Other')
+      }
+    },
+    form: {
+      methods: []
+    }
+  }),
+  created () {
+    const savedMethods = this.ourGroupProfile.paymentMethods || {}
+    const savedMethodsKeys = Object.keys(savedMethods)
+    const savedMethodsCount = savedMethodsKeys.length
+
+    if (savedMethodsCount === 0) {
+      // set the minimum necessary to show the first empty field.
+      Vue.set(this.form.methods, 0, {
+        name: 'choose',
+        value: ''
+      })
+      return
+    }
+
+    for (let index = 0; index < savedMethodsCount; index++) {
+      const method = savedMethodsKeys[index]
+      Vue.set(this.form.methods, index, {
+        name: method,
+        value: this.ourGroupProfile.paymentMethods[method].value
+      })
     }
   },
   computed: {
-    methodsAvailable () {
-      return [
-        {
-          name: 'manual',
-          title: L('Manual'),
-          isAvailable: true,
-          svg: SvgMoney,
-          description: L('Send your contributions using whatever method works best for you.')
-        },
-        {
-          name: 'bitcoin',
-          title: L('Bitcoin'),
-          isAvailable: false,
-          svg: SvgBitcoin,
-          description: L('Coming soon!')
-        }
-      ]
+    ...mapGetters([
+      'ourUsername',
+      'ourGroupProfile'
+    ]),
+    methodsCount () {
+      return Object.keys(this.form.methods).length
+    }
+  },
+  methods: {
+    handleSelectChange (e, index) {
+      // Reset the respective input and focus on it
+      Vue.set(this.form.methods, index, {
+        name: e.target.value,
+        value: ''
+      })
+      Vue.nextTick(() => {
+        this.$refs.fields.childNodes[index].getElementsByTagName('label')[1].focus()
+      })
```

Comment:
Why do we need to wait for the DOM update here?

Reply:
It was needed, but not anymore. nextTick removed!